website/www/site/content/en/get-started/quickstart-java.md

```diff
@@ -283,7 +283,7 @@ We are working on adding the instruction for this runner!
 
 {{< highlight class="runner-spark" >}}
 $ gradle clean execute -DmainClass=org.apache.beam.examples.WordCount \
-    -Dexec.args="--inputFile=pom.xml --output=counts" -Pspark-runner
+    -Dexec.args="--inputFile=sample.txt --output=counts" -Pspark-runner
```

Comment:
I think we should hold off updating the website until a beam release has gone out with the change to add the sample file. The website is published regularly, so if we update it now the quickstart will start telling users to do this with 2.25 and it won't work.

Reply:
Ah, I didn't know that. Okay, I updated `--inputFile` to follow the Python model, so it points to `/path/to/inputfile`. I still think that makes a lot more sense than using the pom as input.